jax/experimental/array_api/_manipulation_functions.py

```diff
@@ -24,9 +24,9 @@ def broadcast_arrays(*arrays: Array) -> list[Array]:
   return jax.numpy.broadcast_arrays(*arrays)
 
 
-def broadcast_to(x: Array, /, shape: tuple[int]) -> Array:
   """Broadcasts an array to a specified shape."""
   return jax.numpy.broadcast_to(x, shape=shape)
+def broadcast_to(x: Array, /, shape: tuple[int, ...]) -> Array:
```

Comment:
The change looks good, but we should move the function definition back to its original location